docs/modules/ROOT/pages/archetype.adoc

```diff
@@ -0,0 +1,142 @@
+there is also an option to supply the values in the same way.
+
+.Archetype Options
+[cols="1a,2a,3a,2a,5a"]
+|===
+| Value | Description | Guidance | Example(s) | Interactive
+
+| ``archetypeVersion``
+| Use an archetype from a specific aiSSEMBLE release
+|
+Typically, developers should target the most recently released archetype. In appropriate circumstances, it is possible to target a pre-release https://maven.apache.org/guides/getting-started/index.html#what-is-a-snapshot-version[snapshot version,role=external,window=_blank] by using the -SNAPSHOT suffix.
+
+|
+* 0.11.0
+* 0.9.8
+* 0.12.0-SNAPSHOT
+| No
+
+| ``artifactId``
+| The folder name as well as module name of your project root
+| A generic description of the project or set of pipelines that will be included in the project. Maven conventions
+suggest the use of a dash to separate terms.
+|
+* ``payment-analytics``
+* ``charge-off-prediction``
+* ``rdec`` (example of a project acronym)
+| Yes
+
+| ``groupId``
+| The namespace in which your Maven modules will live
+| In general, you want it to think about ``groupId`` like the notional reverse ordered internet address of your effort.
+General pattern: ``<internet domain name in reverse>.<your area name>``
+
+| Examples:
+
+* ``com.boozallen.aissemble``
+* ``mil.navy.blueangles.ridealong``
+
+| Yes
+
+| ``package``
+| The package name in which JVM-based source code will be placed
+| Defaults to your ``groupId`` value, which is almost always the right decision, so just select enter to accept this
+default
+| See ``groupId`` examples above
+| Yes
+
+| ``projectDescription``
+| General description of your project
+| Default to "Project that contains aiSSEMBLE compliant pipeline(s)"
+|
+| No
+
+| ``licenseName``
+| The name of the license to use in the project
+| Defaults to Booz Allen's "closed-source-license" but can be updated to a
+https://github.com/boozallen/booz-allen-maven-licenses[Booz Allen license,role=external,window=_blank] or any license
+under mvn license:license-list
+|
+* booz-allen-public-license
+* agpl_v3
+
+| No
+
+| ``projectGitUrl``
+| The git URL of the project
+| The git URL of the project, without the `.git` suffix
+|
+
+
+| Yes
+
+| ``projectName``
+| A short, human-readable version name of the project
+| The human readable version of your ``artifactId``
+|
+
+* aiSSEMBLE
+* NAVY BA Ride Along
+
+| Yes
+
+| ``version``
+| The name of the current version
+| Generally recommended to follow a ``<major version>.<minor version>.<patch version>`` convention. If you don't have a
+strong opinion, start at ``1.0.0-SNAPSHOT``. ``1.0.0-SNAPSHOT`` is the default when instantiating a new project.
+
+Maven has inherent support for "development" versions. This helps projects manage in flight versus released software.
+See <<_snapshot_versions,Snapshot Versions>> in the Additional Considerations section below for more information. Using
```

Comment:
C: The link just circles back to this page and there is no section below when the page is published. I could totally be off base on this but I am not sure this is working as expected.

Reply:
Agree - striking ✅